ui/keyboardMove/src/main.ts

```diff
@@ -70,7 +70,7 @@ type Redraw = () => void;
 
 export function ctrl(root: RootController, step: Step): KeyboardMove {
   let focus = false;
-  let helpModalOpen = false;
+  const helpModalOpen = propWithEffect(false, root.redraw);
```

Comment:
Cool, I was thinking about doing something similar. I believe that we can use `propWithEffect` on `focus`/`setFocus`/`hasFocus` in this controller as well.